yaksh/middleware/user_time_zone.py

```diff
@@ -5,10 +5,10 @@
 
 class TimezoneMiddleware(object):
     """ Middleware to get user's timezone and activate timezone 
-        if user timezone is not available default value 'UTC' is activated """
+        if user timezone is not available default value 'Asia/Kolkata' is activated """
     def process_request(self, request):
         user = request.user
-        user_tz = 'UTC'
+        user_tz = 'Asia/Kolkata'
```

Comment:
Setting timezone to Asia/Kolkata is necessary because it makes easier to set the time for the quizzes. If it kept to UTC then it becomes bit difficult to set quiz timings